modules/gitops-release-notes-1-11-0.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-11-0_{context}"]
+= Release notes for {gitops-title} 1.11.0
+
+{gitops-title} 1.11.0 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="new-features-1-11-0_{context}"]
+== New features
+
+The current release adds the following improvement:
+
+* With this update, you can selectively disable the `redis` and `application-controller` components for an Argo CD instance in a specified namespace. These components are enabled by default.
+To disable a component, set the `enabled` flag to `false` in the `.spec.<component>.enabled` field of the Argo CD Custom Resource (CR). link:https://issues.redhat.com/browse/GITOPS-3723[GITOPS-3723]
++
+.Example: Disabling the `redis` and `application-controller` components
+[source, yaml]
+----
+spec:
+  controller:
+    enabled: false
+  redis:
+    enabled: false
+----
++
+[NOTE]
+====
+This feature is currently limited to the `redis` and `application-controller` components. Support for other components is expected to be included in the forthcoming {gitops-title} v1.11.1 release.
```

Comment:
See Red Hat supplementary style guide for [Future releases or plans](https://redhat-documentation.github.io/supplementary-style-guide/#statements-about-the-future)

> Do not promise that a feature or a fix for a known issue will be included in an upcoming release or according to a specific timeline.

```suggestion
This feature is currently limited to the `redis` and `application-controller` components. It is expected that support for other components will be included in a future {gitops-title} release.
```